src/lib/ThumbnailsSidebar.js

```diff
@@ -120,9 +174,16 @@ class ThumbnailsSidebar {
      */
     createPlaceholderThumbnail(itemIndex) {
         const thumbnailEl = document.createElement('button');
-        thumbnailEl.className = 'bp-thumbnail';
+        const pageNum = itemIndex + 1;
+
+        thumbnailEl.className = CLASS_BOX_PREVIEW_THUMBNAIL;
         thumbnailEl.setAttribute('type', 'button');
-        thumbnailEl.appendChild(this.createPageNumber(itemIndex + 1));
+        thumbnailEl.dataset.bpPageNum = pageNum;
+        thumbnailEl.appendChild(this.createPageNumber(pageNum));
+
+        if (pageNum === this.currentPage) {
+            thumbnailEl.classList.add('bp-thumbnail-is-selected');
```

Comment:
You're using constants for selectors like this in other places

Reply:
I just made constants for reoccurring strings, but I can make them all constants

Reply:
fine by me!